about.html

```diff
@@ -0,0 +1,100 @@
+           
+                <span>Thank you for taking the time to learn about us. We're here to make your shopping experience delightful and memorable. If you have any questions, need assistance, or simply want to share your feedback, please don't hesitate to contact our friendly team. <br> <a href="">lawmsangivarte22@gmail.com</a></span>
+                
+            </div>
+        </section>
+
+        <section class="contact">
+            <div class="contact-container">
+                <h2>Unleash your creativity and share your own designs with us - join the magic of personalization!</h2>
+                <div class="line"></div>
+                <p>To proceed with sharing your own designs, simply reach out to our dedicated design team through our contact form or email. We'll work closely with you to bring your vision to life, ensuring every detail is perfected. Join us in creating unique and unforgettable personalized gifts!</p>
+                <img src="images/9.jpeg" alt="img">
+            </div>
+
+            <div class="contact-container">
+                <h2>Embrace the opportunity to monetize your artistic flair and captivate a wider audience by selling your own uniquely crafted designs with us.</h2>
+                <div class="line"></div>
+                <p>To embark on your journey of selling your own designs, simply submit your portfolio and design samples to our dedicated team via our online submission form. We'll carefully review your work and get in touch with you to discuss collaboration opportunities. Let your creative vision thrive and turn your passion into a flourishing business with our support and platform.</p>
+                <div class="image-container">
+                    <div class="contact-image">
+                        <img src="images/7.jpeg" alt="shirt">
+                    </div>
+                    <div class="contact-image1">
+                        <img src="images/8.jpeg" alt="shirt">
+                    </div>
+                </div>
+            </div>
+        </section>
+
+         <section class="section-partner">
+        <div class="partner-container">
+            <div class="partner-header">
+                    <h2>Partner</h2>
+                    <div class="partner-line"></div>
+            </div>
+            <div class="partner-content">
+                <img src="images/amazon.png" alt="amazon">
+                <img src="images/hm-logo.png" alt="h&m">
+                <img src="images/adidas.png" alt="adidas">
+                <img src="images/shopify.png" alt="shopify">
+                <img src="images/flipkart.png" alt="flipkart">
+            </div>
+        </div>
```

Comment:
- [ ] You have done great creating the partner section and styling it according to the Given design. :+1: :+1: 
- -  Looking at this from the user's perspective, the logos are not visible here, these are black logos in a dark background and therefore make it hard to see, I'll suggest you modify the color of these logos to ones similar in the design. :smile: :+1:  

- - Your current design:
![Screenshot from 2023-06-28 10-12-22](https://github.com/Lawmsangi/Module-One-Capstone/assets/107632780/5e2093c5-9c94-49e4-852b-107976e9e768)